### PR TITLE
File manager overlay within Edit mode supports upload of files by …

### PR DESCRIPTION
**Pull request by leraradinovich**
…dragging them onto the window but it doesn’t work reliably. Progress bar appears but it never completes #4059

@squallstar

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -62,36 +62,39 @@ function parseThumbnail(file) {
 
 function navigateToDefaultFolder() {
   if (typeof data === 'undefined' || !data || !data.appId) {
-    // No folder was specified
```
Comment (squallstar) [inline js/interface.js]: why did you change the indentation on these comments?

File: js/interface.js [hunk 2]
```diff
@@ -62,36 +62,39 @@ function parseThumbnail(file) {
...
   var $el = $('[data-app-id="' + data.appId + '"][data-browse-folder]');
 
-  // Activate folder on left sidebar
+// Activate folder on left sidebar
   if ($el.data('type') === 'organization') {
     $listHolder = $el;
   } else {
     $listHolder = $el.find('.list-holder');
   }
-
```
Comment (squallstar) [inline js/interface.js]: Please set up your editor to trim empty spaces

File: js/interface.js [hunk 3]
```diff
@@ -62,36 +62,39 @@ function parseThumbnail(file) {
...
+  
   $('.dropdown-menu-holder').find('.list-holder.active').removeClass('active');
   $listHolder.first().addClass('active');
 
-  // Set first folder of breadcrumbs
+// Set first folder of breadcrumbs
   resetUpTo($el);
-
+  
+  if(data.appId) {
```
Comment (squallstar) [inline js/interface.js]: Did you check the code guidelines? There should be a space after the `if` keyword.

**Comment by squallstar** [comment]
ref https://github.com/Fliplet/fliplet-studio/issues/4059

File: js/interface.js (hunk 1 above)
Comment (leraradinovich) [reply js/interface.js]: done

File: js/interface.js (hunk 3 above)
Comment (leraradinovich) [reply js/interface.js]: done
